spam/02_spam_data_augmentation_tutorial.py

```diff
@@ -271,16 +287,18 @@ def replace_adjective_with_synonym(x):
 print(f"Augmented training set size: {len(df_train_augmented)}")
 
 # %% [markdown]
-# We have almost doubled our dataset using TFs! Note that despite `n_per_original` being set to 2, our dataset may not exactly triple in size, because sometimes TFs return `None` instead of a new example (e.g. `change_person` when applied to a sentence with no persons).
+# We have almost doubled our dataset using TFs!
+# Note that despite `n_per_original` being set to 2, our dataset may not exactly triple in size, because sometimes TFs return `None` instead of a new example (e.g. `change_person` when applied to a sentence with no persons).
```

Comment:
This is actually an interesting point... should we be making sure that each data point gets copied exactly the same number of times?  I think this is a more proper way of doing this... should discuss IRL